.github/workflows/publish-debug-image.yaml

```diff
@@ -7,6 +7,7 @@ on:
 
 env:
   DOCKER_REPOSITORY: "tigrisdata/tigris-debug"
+  QUAY_IMAGE: "quay.io/tigrisdata/tigris-debug"
```

Comment:
Isn't this a repository instead of an image?

Reply:
I guess I see why we used image here from L49-L51. To avoid confusion, should we make the DOCKER_REPOSITORY variable similar (e.g. DOCKER_IMAGE) or vica versa?